Documentation/upgrade.md

```diff
@@ -127,7 +127,7 @@ Any pod that is using a Rook volume should also remain healthy:
 ## Upgrade Process
 The general flow of the upgrade process will be to upgrade the version of a Rook pod, verify the pod is running with the new version, then verify that the overall cluster health is still in a good state.
 
-In this guide, we will be upgrading a live Rook cluster running `v0.5.0` to the next available version of `v0.5.1`.
+In this guide, we will be upgrading a live Rook cluster running `v0.6.0` to the next available version of `master`.
```

Comment:
In this doc, im upgrading to `master` since there is no newer version released.

Reply:
what if we use v0.6.2 in the example?

Reply:
You mean from v0.6.2 to master or v0.5.x to v0.6.2?

If the latter, then everything should work fine. The addition that broke the upgrade were added after 0.6.2 with the remove of the version-tag and the ownership feature

Reply:
yes, the common case is likely from v0.6.2